### PR TITLE
[Compiling Swift Generics Book] Fixing some typos

### PR DESCRIPTION
**Pull request by AliSoftware**
While I'm reading @slavapestov 's PDF from https://forums.swift.org/t/compiling-swift-generics-part-i/60898, I've noticed a couple of small typos in the book, so this PR aims to fix the ones I found.

---

I'm still at the very beginning of my reading, so not much fixes so far. I'll probably add more commits to that PR as I find more typos down the road, hence why I'll keep this as draft for the moment.

_Besides, I'm reading the PDF from my iPad and thus making those changes — and submitting them — from the GitHub iPad app… which might not be the easiest and nicest way to navigate and edit the large `.tex` file 😅  (I'll likely add the future commits / typo fixes from a computer with better way to navigate said `.tex` file 😛 )… but I still wanted to start submitting these first 3 ones early, before I forgot and failed to find them again by the time I'll have access to a Mac tomorrow_ 😉

### CODE REVIEW

File: docs/Generics/generics.tex [hunk 1]
```diff
@@ -161,7 +161,7 @@ \chapter{Introduction}\label{roadmap}
 
 An important part of compiler implementation is the design of domain objects to model concepts in the language being compiled. One way to think of a compiler is that it is \emph{a library for implementing the target language}. A well-designed set of domain objects facilitates the introduction of new language features that compose existing functionality in new ways.
 
-The generics implementation deals with four fundamental domain objects: \emph{generic signatures}, \emph{substitution maps}, \emph{requirement signatures}, and \emph{conformances}. As you will see, they are defined as much by their inherent structure, as their relationship with each other. Subsequent chapters will dive into all the details, but first, we're going to look at a series of worked examples to help you understand the big picture.
```
Comment (slavapestov) [inline docs/Generics/generics.tex:164]: "worked example" was intentional here

**Review by slavapestov — APPROVED** [review]

**Comment by slavapestov** [comment]
Thanks!

File: docs/Generics/generics.tex (hunk 1 above)
Comment (AliSoftware) [reply docs/Generics/generics.tex:164]: Ah, TIL this was a phrase in English! (I'm not a native speaker)

Comment (AliSoftware) [reply docs/Generics/generics.tex]: Reverted in https://github.com/apple/swift/pull/61604/commits/a509c7332b450c23ad2a5e0f0310742376cf914b

**Comment by AliSoftware** [comment]
Ok, I'm not sure I'll have time to finish my reading of the whole book as soon as I thought after all, so we might as well merge the typo fixes so far and I could always open a subsequent PR when I get time to resume reading and if I find more typos to fix then 😉 

@slavapestov since I've added more commits since your initial review & approval, feel free to double-check the new ones I've added since, then merge the PR if you're ok with them all 🙂

**Comment by slavapestov** [comment]
@swift-ci Please smoke test and merge

**Comment by AliSoftware** [comment]
Wow, a reply only 26s after being ping'd, impressive… I'm not sure who I'd bet on in a race between you and Noby 🏇 😄

**Comment by AliSoftware** [comment]
@slavapestov seems CI failed and this this was never merged after all 😅

**Comment by AnthonyLatsis** [comment]
@AliSoftware Could you clean up the commit history?

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test 

Thanks!

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Comment by AliSoftware** [comment]
I took a look at the suggestions from this PR vs the latest content of the `.tex` file and most of the content have changed or be reworded since (couldn't find the sentences I fixed typos on in the latest `main` at least), so I think it's ok to close 👍

**Pull request closed without merging**